src/en/juju-backups.md

```diff
@@ -0,0 +1,405 @@
+Title: Back up and restore Juju
```

Comment:
Recommend consistency when introducing commands. You only sometimes employ the "usage:" tag.